pkg/mkverification-raw-efi/vmcap

```diff
@@ -0,0 +1,298 @@
+#!/usr/bin/python3
+#
+# tool for querying VMX capabilities
+#
+# Copyright 2009-2010 Red Hat, Inc.
+#
+# Authors:
+#  Avi Kivity <avi@redhat.com>
+#
+# This work is licensed under the terms of the GNU GPL, version 2.  See
```

Comment:
Is this part of the Linux kernel? Or a file by itself? We want to limit ourselves to standard, well-known GPL code like the kernel etc.

What information do we need? Can we just read and print binary values (assuming the MSR bit definitions are standard they can be interpreted to text somewhere else and not in EVE).

Reply:
Thank you for noticing that. I will write a script that replaces vmcap and produces the same results.